alerts/plugins/etintel.py

```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+import os
+from configlib import getConfig, OptionParser
+import sys
+from datetime import datetime, timedelta
+sys.path.append(os.path.join(os.path.dirname(__file__), "../lib"))
```

Comment:
We're trying to get away from modifying the sys.path if possible, we can actually do something like:
`from lib.etintellib import etintel`